default.nix

```diff
@@ -73,44 +73,48 @@ rec {
           echo "Migrating home-manager installation..."
           if [[ -r "$HOME/.config/nixpkgs/home.nix" ]]; then
             ${pkgs.patch}/bin/patch --no-backup-if-mismatch $HOME/.config/nixpkgs/nix-on-droid.nix ${pkgs.writeText "patch" ''
-              @@ -27,15 +27,9 @@
+              @@ -30,18 +30,8 @@
                  # Read the changelog before changing this value
-                 system.stateVersion = "20.03";
+                 system.stateVersion = "20.09";
 
               -  # After installing home-manager channel like
-              -  #   nix-channel --add https://github.com/rycee/home-manager/archive/release-20.03.tar.gz home-manager
+              -  #   nix-channel --add https://github.com/rycee/home-manager/archive/release-20.09.tar.gz home-manager
               -  #   nix-channel --update
               -  # you can configure home-manager in here like
               -  #home-manager.config =
               -  #  { pkgs, ... }:
               -  #  {
+              -  #    # Read the changelog before changing this value
+              -  #    system.stateVersion = "20.09";
```

Comment:
What is this one? Did you mean home.stateVersion?

Reply:
Oh, you are right, I meant home.stateVersion :) You can change it if you like, otherwise I will do it in the next days